sass/components/_files.scss

```diff
@@ -168,7 +168,6 @@
         .image-loaded {
             @include single-transition(all, .1s, linear);
             @include border-radius(4px);
-            border: 1px solid transparent;
```

Comment:
Deleted this border since it makes a slight scroll pop when applied.

Reply:
Sorry, we can't really delete this. Is the scroll pop caused by a 1px border really that noticable?

Reply:
Yes, I think roughly 3-5px, it varies and it's not consistent depending on whether the image is at max-height of 350px. I suggest not to have it at the moment.